components/learn_more_trial_modal/learn_more_trial_modal.tsx

```diff
@@ -0,0 +1,137 @@
+    onExited: () => void;
+}
+
+const LearnMoreTrialModal = (
+    {
+        onClose,
+        onExited,
+    }: Props): JSX.Element | null => {
+    const {formatMessage} = useIntl();
+
+    const show = useSelector((state: GlobalState) => isModalOpen(state, ModalIdentifiers.LEARN_MORE_TRIAL_MODAL));
+
+    if (!show) {
+        return null;
+    }
+
+    useEffect(() => {
+        trackEvent(
+            TELEMETRY_CATEGORIES.SELF_HOSTED_START_TRIAL_MODAL,
+            'learn_more_trial_modal_post_enterprise_view',
+        );
+    }, []);
+
+    const steps: LearnMoreTrialModalStepProps[] = useMemo(() => [
+        {
+            id: 'guestAccess',
+            title: formatMessage({id: 'trial_benefits.modal.guestAccessTitle', defaultMessage: 'Determine user access with Guest Accounts'}),
+            description: formatMessage({id: 'trial_benefits.modal.guestAccessDescription', defaultMessage: 'Collaborate with users outside of your organization while tightly controlling their access channels and team members.'}),
+            svgWrapperClassName: 'guestAccessSvg',
+            svgElement: (
+                <GuestAccessSvg
+                    width={400}
+                    height={180}
+                />
+            ),
+            buttonLabel: formatMessage({id: 'benefits_trial.modal.learnMore', defaultMessage: 'Learn More New!'}),
+        },
+        {
+            id: 'complianceExport',
+            title: formatMessage({id: 'trial_benefits.modal.complianceExportTitle', defaultMessage: 'Automate compliance exports'}),
+            description: formatMessage({id: 'trial_benefits.modal.complianceExportDescription', defaultMessage: 'Run daily compliance reports and export them to a variety of formats consumable by third-party integration tools such as Smarsh (Actiance).'}),
+            svgWrapperClassName: 'personMacSvg',
+            svgElement: (
+                <PersonMacSvg
+                    width={400}
+                    height={180}
+                />
+            ),
+            buttonLabel: formatMessage({id: 'benefits_trial.modal.learnMore', defaultMessage: 'Learn More'}),
+        },
+        {
+            id: 'pushNotificationService',
+            title: formatMessage({id: 'trial_benefits.modal.pushNotificationServiceTitle', defaultMessage: 'Mobile Secure-ID Push Notifications'}),
+            description: formatMessage({id: 'trial_benefits.modal.pushNotificationServiceDescription', defaultMessage: 'ID-only push notification setting offers a high level of privacy while still allowing your team members to benefit from mobile push notifications.'}),
+            svgWrapperClassName: 'personBoxSvg',
+            svgElement: (
+                <PushNotificationsSvg
+                    width={400}
+                    height={200}
+                />
+            ),
+            buttonLabel: formatMessage({id: 'benefits_trial.modal.learnMore', defaultMessage: 'Learn More'}),
+        },
+    ], []);
```

Comment:
In a recent change (https://github.com/mattermost/mattermost-webapp/pull/9803) a new step was added for Playbooks. Do we need to add it here as well?